node/cn/handler_test.go

```diff
@@ -380,6 +380,105 @@ func TestBroadcastTxsFrom_DefaultCase(t *testing.T) {
 	pm.BroadcastTxs(txs)
 }
 
+func TestReBroadcastTxs_CN(t *testing.T) {
+	pm := &ProtocolManager{}
+	pm.nodetype = node.CONSENSUSNODE
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+
+	peers := newPeerSet()
+	pm.peers = peers
+	createAndRegisterPeers(mockCtrl, peers)
+
+	pm.ReBroadcastTxs(txs)
+}
+
+func TestReBroadcastTxs_PN(t *testing.T) {
+	// CN Peer=0, PN Peer=1
+	{
+		pm := &ProtocolManager{}
+		pm.nodetype = node.PROXYNODE
+		mockCtrl := gomock.NewController(t)
+
+		peers := newPeerSet()
+		pm.peers = peers
+
+		enPeer := NewMockPeer(mockCtrl)
+		enPeer.EXPECT().ConnType().Return(p2p.ConnType(node.PROXYNODE)).Times(2)
+		enPeer.EXPECT().SendTransactions(gomock.Any()).Times(1)
```

Comment:
How about checking the argument value for `SendTransactions`?

Reply:
Reflected!